### PR TITLE
elaborate on infrastructure options

### PR DESCRIPTION
**Pull request by afeld**
...particularly around static sites.

/cc @toolness @jezhumble @NoahKunin @thecapacity @maya @leahbannon

### CODE REVIEW

File: _pages/infrastructure/aws.md [hunk 1]
```diff
@@ -41,26 +41,16 @@ These are just the concepts necessary for initial on-boarding. AWS has an [exten
 
 There are a few special notes on using any IaaS in the Federal context.
 
+### Sandbox
+
+See @jezhumble's [Slack post](https://18f.slack.com/files/jezhumble/F18GU8ABW/AWS_Sandboxes_for_18F_and_PIF_use) about the 18F sandbox account.
```
Comment (afeld) [inline _pages/infrastructure/aws.md]: This page needs a rewrite (#40), but put this in as a placeholder.

**Comment by toolness** [comment]
This looks great, thanks @afeld!

Would it help to specifically point out that deploying to gh-pages for internal sites is *not* a recommended option? I'd simply assumed it would be, and I wonder if others might too... Also, I've found [outdated videos](https://youtu.be/ZRhRU5y0jEk?t=35m47s) that show 18F content hosted on github pages, so some folks may see those and mistakenly think it's ok, too.

File: _pages/infrastructure.md [hunk 2]
```diff
@@ -12,6 +18,37 @@ title: Infrastructure
...
+See cloud.gov page on [deploying static sites](https://docs.cloud.gov/apps/static/).
+
+* **Internal:** free
+* **External:** see the [pricing page](https://docs.cloud.gov/intro/pricing/pricing-model/)
+
+#### Federalist
+
+See [the Federalist homepage](https://federalist.18f.gov) for more information.
+
+* **Internal:** free
```
Comment (wslack) [inline _pages/infrastructure.md]: not sure about this <3

We should word this clearly to say that Federalist support of a paid-for project should be paid for just like any other Federalist site.

Comment (afeld) [inline _pages/infrastructure.md]: Are you thinking for things like Great Pitch projects? Maybe changing the definition of "internal" above to be "paid for out of 18F overhead"? I defer to you on this!

**Comment by afeld** [comment]
@toolness Good call, thanks! Done.

File: _pages/infrastructure.md (hunk 2 above)
Comment (wslack) [inline _pages/infrastructure.md]: That makes more sense. Editing.

Comment (wslack) [inline _pages/infrastructure.md]: Hmmm. Wording is hard. Accepting ideas.

Comment (ctro) [inline _pages/infrastructure.md]: @wslack I was gonna merge, but looks like you sill have a nit to pick here?

Comment (wslack) [inline _pages/infrastructure.md]: Yeah... @ctro it's the nuance that if you need to use cloud.gov for a project that people are paying for, cloud.gov needs its own IAA with the partner to recover costs, and some "internal" projects are paid for by OSCIT anyways. Not sure how to explain that clearly.

Comment (afeld) [inline _pages/infrastructure.md]: Should we just punt on this and say "ask in #federalist"?

Comment (wslack) [inline _pages/infrastructure.md]: It's not just Federalist, its also cloud.gov, pages, or anything else we're spending money on. Maybe for all of the internal categories we can put "Likely free, but check with #owner channel with your use case."

Thoughts?

Comment (afeld) [inline _pages/infrastructure.md]: Good enough for now 👍

**Comment by afeld** [comment]
Good to merge?

**Comment by wslack** [comment]
👍  by me

**Pull request merged**